base/src/org/compiere/util/EMail.java

```diff
@@ -1483,4 +1483,37 @@ public static void main (String[] args)
 		email.send();
 	}   //  main
 
+	public static String validateMailDelivery(Properties ctx, String requestEMail, String requestPassword, int eMailConfigId, String name) {
+		if (requestEMail == null || requestEMail.length() == 0)
+			return "No Request EMail for " + name;
+		//
+		EMail email = new EMail(MClient.get(ctx), eMailConfigId, requestEMail, requestEMail,
+				"Adempiere EMail Test",
+				"Adempiere EMail Test", false);
+		email.createAuthenticator (requestEMail, requestPassword);
+
+		try
+		{
+			String msg = email.send();

```

Comment:
@rcapecce Please use natural language to name the variables that makes it easier to read and analyze the code, change msg to message

Reply:
@e-Evolution I change this variable name